tests/models/test_models.py

```diff
@@ -26,19 +26,21 @@
 @pytest.mark.parametrize("model", MODELS)
 @pytest.mark.parametrize("dtype", ["float"])
 @pytest.mark.parametrize("max_tokens", [128])
+@pytest.mark.parametrize("enforce_eager", [False, True])
 def test_models(
     hf_runner,
     vllm_runner,
     example_prompts,
     model: str,
     dtype: str,
     max_tokens: int,
+    enforce_eager: bool,
 ) -> None:
     hf_model = hf_runner(model, dtype=dtype)
     hf_outputs = hf_model.generate_greedy(example_prompts, max_tokens)
     del hf_model
 
-    vllm_model = vllm_runner(model, dtype=dtype)
+    vllm_model = vllm_runner(model, dtype=dtype, enforce_eager=enforce_eager)
```

Comment:
Reminder to revert

Reply:
do we have to revert this? I think it is better testing both cases here? I also haven't found any other test that verifies cuda graph works correctly (lmk if there is a test)

Reply:
Yes, makes sense, the only concern I had was that this could lead to OOM in CI, but as long as it is working it is definitely useful to have this test. By the way, should we add some tests to validate correctness for chunked prefills and hybrid batches?

Reply:
Got it. Alternatively, we can have another test that just checks this with a single model. I will make another PR for this.